Secure-by-design-guide.md

```
@@ -0,0 +1,548 @@
+```
+
+#### CIA-T：4つの古典的なセキュリティ関心事
+
+情報セキュリティの世界では伝統的に「CIA」というモデルが使われますが、本書ではさらに「T（追跡可能性）」を加えた「CIA-T」を紹介しています。
+
+| 頭文字 | 意味 | 説明 | 具体例 |
+|---|---|---|---|
+| **C**onfidentiality（機密性） | 秘密にすべき情報を秘密のまま保つこと | 診療記録が第三者に漏れないこと |
+| **I**ntegrity（完全性） | データが許可された方法でしか変更されないこと | 選挙の投票結果が改ざんされていないこと |
+| **A**vailability（可用性） | 必要なときにデータ・機能が使えること | 消防が火災発生場所の情報に即座にアクセスできること |
+| **T**raceability（追跡可能性） | 誰が・いつ・何を変更/参照したかを追跡できること | GDPR（EU一般データ保護規則）が求める監査ログ |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)